core/src/main/java/com/kin/ecosystem/core/accountmanager/AccountManagerImpl.java

```diff
@@ -116,19 +119,7 @@ private void setAccountState(@AccountState final int accountState) {
 				case REQUIRE_CREATION:
 					eventLogger.send(StellarAccountCreationRequested.create());
 					Logger.log(new Log().withTag(TAG).put("setAccountState", "REQUIRE_CREATION"));
-					// Trigger account creation from server side.
-					authRepository.getAuthToken(new KinCallback<AuthToken>() {
-						@Override
-						public void onResponse(AuthToken response) {
-							setAccountState(PENDING_CREATION);
-						}
-
-						@Override
-						public void onFailure(KinEcosystemException error) {
-							instance.error = error;
-							setAccountState(ERROR);
-						}
-					});
+					setAccountState(PENDING_CREATION);
```

Comment:
for safety check getKinAccount()!= null in line 131

Reply:
👍